### PR TITLE
PieChart: Added attributedLabel

### PR DESCRIPTION
**Pull request by marbetschar**
First of all: This library is AMAZING!! Thank you so much for all of your efforts, it's just astonishing what is possible with it! After playing around with it, I was able to solve nearly everything I wanted to do.

The only thing I was missing, was the ability to use `NSAttributedString` as described in https://github.com/danielgindi/Charts/issues/2182 - that's why I've added this ability into this pull request. Would be awesome to see this addition in the official release!

Basically this enables us to enrich labels in PieCharts - for example to display the actual value of the `PieChartDataEntry`, or every label can be colored like the slice itself etc.

Here you can see a live example of what I've done with it:

bildschirmfoto 2017-02-23 um 01 28 46

As said, It would be awesome to see this feature addition in the official release.

### CODE REVIEW

**Comment by marbetschar** [comment]
Additionally added the ability to use NSAttributedString in any AxisValueFormatter.

**Comment by marbetschar** [comment]
What has to be done to see this merged into the official release?

**Comment by liuxuan30** [comment]
Have  to wait for @danielgindi review :( and he seems busy recently.

**Comment by marbetschar** [comment]
@danielgindi what can we do to help? I know that maintaining a lib like this is quite a task and we all get too busy for doing open source efforts from time to times.

**Comment by liuxuan30** [comment]
@marbetschar when adopting new features, we also need to consider android portion, but it's too much to ask for a contributor for both android and iOS, so Daniel usually takes the work for android.

You have been helping a lot, but I guess you just have to wait.

**Comment by marbetschar** [comment]
any news on this one?

**Comment by marbetschar** [comment]
anything?

**Comment by jjatie** [comment]
@marbetschar Are you able to rebase on master so we can get this merged?

**Comment by felipemobile** [comment]
Hi @danielgindi any news on this one?

**Comment by liuxuan30** [comment]
have to delay this to 4.0, if @marbetschar you can rebase it on our 4.0 branch, or wait 4.0 branch is merged into master.

4.0 branch currently is holding all swift refactoring PRs, so many things changed, in terms of more swift-y. I'd expect many refactoring on the PRs as well. Sorry for any inconvenience.
